### ISSUE TITLE
Make it faster to include all possible variables from a sensor

### ISSUE THREAD

**Issue by SRGDamia1** [ODM2/ODM2WebSDL#33]
For sensors commonly recording multiple variables, make it faster for the user to register that sensor in its most common configuration.  So, for a Decagon CTD, have a shorter way to simultaneously register the same sensor with all three variables it can record.

**Comment by horsburgh** [comment]
@SRGDamia1 - because the user must select each variable, its units, and input the UUID for each time series, we need to back-burner this one. Also - users may choose not to record all three variables.

If we want to revisit later, we can, but we've got a huge amount of functionality to add yet, and we need to focus on things that aren't working at all yet. There is already a functional way to do this.

**Comment by SRGDamia1** [comment]
Not a problem; the back-burner is fine.  I just want it to be on the wish list.

**Comment by SRGDamia1** [comment]
There has *got* to be a faster way to add all the sensor data.  I'm creating a bunch of sites for stations we're giving away and it feels like a thousand clicks to set them up.

**Comment by SRGDamia1** [comment]
+1 on this again.  I've begun to dread being handed a bunch of boards to set up on the portal and program because it takes so. much. clicking.  And then I have to copy and paste the UUID's one-by-one.  It's awful.

**Comment by mbarneytu** [comment]
Agreed. I was about to submit a similar request: we'd love to be able to clone a site. 95% of our sites will have an identical configuration of sensors and variables. Only the site name and location will change. So being able to clone an existing site to a new site would be a huge time savings as we add sites to our monitoring network.
